var/spack/repos/builtin/packages/py-numpy/package.py

```diff
@@ -106,8 +106,8 @@ class PyNumpy(PythonPackage):
     depends_on("py-tomli@1:", when="@1.26: ^python@:3.10", type="build")
     depends_on("py-setuptools@60:", when="@1.26: ^python@3.12:", type="build")
     # https://github.com/spack/spack/pull/32078
-    depends_on("py-setuptools@:63", when="@:1.25", type=("build", "run"))
-    depends_on("py-setuptools@:59", when="@:1.22.1", type=("build", "run"))
+    depends_on("py-setuptools@:63", when="@:1.25", type="build")
+    depends_on("py-setuptools@:59", when="@:1.22.1", type="build")
```

Comment:
Is setuptools required by `numpy.distutils`?

Reply:
No, only `distutils` is a hard requirement. There's a bunch of conditional code like:
```python
if 'setuptools' in sys.modules:
```
but I don't think that that does anything especially relevant.

Reply:
Does `numpy.distutils` behave any differently when setuptools is installed vs. falling back on distutils? Sometimes setuptools fixes bugs in distutils.